Lib/test/test_tempfile.py

```diff
@@ -1864,6 +1960,11 @@ def test_flags(self):
         finally:
             os_helper.unlink(filename)
 
+    @unittest.skipUnless(hasattr(os, 'chflags'), 'requires os.chflags')
+    def test_flags(self):
+        flags = stat.UF_IMMUTABLE | stat.UF_NOUNLINK
+        self.check_flags()
```

Comment:
You can pass  flags to check_flags(), rather than having a copy inside the function. I also suggest moving the comment here:

```suggestion
        # skip the test if these flags are not supported (ex: FreeBSD 13)
        flags = stat.UF_IMMUTABLE | stat.UF_NOUNLINK
        self.check_flags(flags)
```

Reply:
I do not want to repeat the same comment 3 times.